src/Components/Views/CreateChallenge/CreateChallenge.js

```diff
@@ -13,12 +13,34 @@ function CreateChallenge(props) {
     }
 
     
-    let [tests, setTests] = useState([{value1: null, value2: null, value3: null}])
+    let [tests, setTests] = useState([{value1: "", value2: "", value3: ""}])
```

Comment:
Please be more semantic with these variable names.

Something like.. description, arguments, and expectedValue